__tests__/plugin.test.js

```diff
@@ -16,6 +16,33 @@ describe('babel plugin', () => {
     process.env.REANIMATED_PLUGIN_TESTS = 'jest';
   });
 
+  it('injects its version', () => {
+    const input = `
+      function foo() {
+        '__Reanimated Babel Plugin version injection entry point';
+        var foo = 'bar';
+      }
+    `;
+
+    const { code } = runPlugin(input, {});
+    expect(code).toContain('global._REANIMATED_VERSION_BABEL_PLUGIN = "');
```

Comment:
Do you think we can read the version from `package.json` here using `fs.readFileSync` and match the whole line, i.e.
```suggestion
    const version = // ...
    expect(code).toContain(`global._REANIMATED_VERSION_BABEL_PLUGIN = "${version}";`);
```

Reply:
Got it.